orga/app/styles/globals/colors.scss

```
@@ -10,6 +10,7 @@ $white: #ffffff;
 $blue-hover: #3257D9;
 $dark-blue-pro: #1A38A0;
 $dark-green-certif: #1A8C89;
+$dark-green-certif-light: rgba(26, 140, 137, 0.1);
```

ici on doit pouvoir faire ?
```suggestion
$dark-green-certif-light: lighten(#1A8C89, 0.9);
```

---

Voir même ;)
```suggestion
$dark-green-certif-light: lighten($dark-green-certif, 0.9);
```

---

done